frontend/src/components/supp-allowances/TransportationAllowance.vue

```diff
@@ -146,6 +146,33 @@
           </v-card>
         </v-col>
       </v-row>
+
+      <v-row v-if="showBackDatePicker" class="pa-7 pt-4">
+        <v-col cols="12" md="6">
+          <AppLabel>Transportation Allowance Start Date</AppLabel>
+          <p>
+            Please provide the date your centre began to safely transport children between your centre and school (or a collective point of access) to support regular ongoing child care. Please note,
+            the start date cannot be before the start of the Funding Agreement.
+          </p>
+        </v-col>
+        <v-col cols="12" sm="6" lg="5" class="ml-0">
+          <AppDateInput
+            min-width="250px"
```

Comment:
could we move v-model to the top? also, I think some of these properties are redundant: type, locale, hide-actions

Reply:
done!